### ISSUE TITLE
Research localization

### ISSUE THREAD

**Issue by Siliconlad** [automat-ed/autonomous#5]
For Demo 2, we need to be able to show that the robot can determine where it is located within the simulation. Before we can do this, we need to conduct some research into how to do this. In addition, we need to compile a list of required sensors to pass to the hardware team so that they can design the robot.

# Task
To conduct research into different methods for localizing the robot in simulation as well as how to create and utilize a preconfigured map of the environment. All research should be documented in the [Google Drive folder](https://drive.google.com/drive/folders/1m9Mdte_oveNHa0qymPkkvpYUwnPCyi6p?usp=sharing). In addition, a list of required sensors should be compile along with their position for optimal use.

**Comment by Siliconlad** [comment]
The robot navigation expert provided some useful advice on how to tackle this problem. He strongly recommends us to take a look at the ROS navigation package as it implements all the algorithms that we need. He offered to help convert top-down view of our environment in simulation into a map that can be used in the ROS navigation package. This will be worth following up on in a timely manner.

See the [Drive document for the meeting minutes and questions asked](https://docs.google.com/document/d/1XrZ1vRc6M1mgdpx8cbclVnf0KHoxaUY7dmiIMC2b2r0/edit?usp=sharing).

**Comment by Siliconlad** [comment]
Superseded by #11 and #12.
